src/mame/drivers/seattle.cpp

```diff
@@ -133,7 +133,23 @@
         P16 is a 6 pin marked "Aux in"
         P3 is a 6 pin marked "Bill in"
         P8 is a 14 pin marked "Aux Latched Outputs"
-        P22 is a 9 pin marked "serial port"
+        P22 is a 9 pin marked "serial port" used for bill acceptors and Blitz
+            '99's memory card reader (leads to the IOASIC's serial port).
+***************************************************************************
+
+    NFL Blitz '99 has an addon card reader board for saving custom stats and
+    plays through an N64 controller pak. One can save custom stats through
+    arcade Blitz '99 to use with N64 NFL Blitz or vice versa. NFL Blitz 2000
+    Gold does not support the card reader.
+    
+    5772-15870-03 Joyport
+        U1: ADM3202ARN
+        U2: ATMEL AT90S4414 AVR based MCU labeled 'U2 A22799.1 VER 2.0' (PLCC44)
+        U5: UCN5804LB
+        U6: MM74HC574WM
+        Y1: 4.000 XTAL
+
+    TODO: Not supported
```

Comment:
Doesn't conform to the header QA standard: should go on top, near the other item about Carnevil.
Arguably both notes should be expressed with romset name rather than full name, along the lines of `blitz99: unemulated N64 controller pak option;`.

Reply:
Consider it done.